geowombat/ml/classifiers.py

```diff
@@ -5,12 +5,14 @@
 
 import xarray as xr
 from sklearn.pipeline import Pipeline
-from sklearn.model_selection import GridSearchCV, GroupShuffleSplit
 from sklearn.preprocessing import LabelEncoder
 from sklearn_xarray import wrap, Target
```

Comment:
Is `sklearn_xarray` still being developed?

Reply:
Good question. Doesn't seem to be actively developed.  Should we try to bring some of those functions into geowombat?